GOVERNANCE.md

```diff
@@ -1,88 +1,234 @@
+
+### Becoming an Organization Member
+
+Open an issue towards [the community repository](https://github.com/backstage/community) using the [org membership request template](#TODO).
+
+### Privileges
+
+- Membership in the Backstage GitHub organization
+
+## Plugin Maintainer
+
+A Plugin Maintainer is responsible for maintaining an individual Backstage plugin or module. This includes reviewing contributions and responding to issues towards an individual plugin, as well as keeping the plugin up to date.
+
+Plugin Maintainer is a lightweight form of ownership that is primarily reflected though code owners of the plugin packages in the [CODEOWNERS](./.github/CODEOWNERS) file. Each plugin can have one or more maintainers. If a plugin becomes a significant part of the Backstage ecosystem, it may be promoted to be a distinct project area instead.
+
+A Plugin Maintainer has all the rights and responsibilities of an Organization Member.
+
+### Responsibilities
+
+- Review the majority of PRs towards the plugin
+- Respond to GitHub issues related to the plugin
+- Keep the plugin up-to-date with Backstage libraries and other dependencies
+- Follow the [reviewing guide](REVIEWING.md)
+
+### Requirements
+
+- Is an Organization Member
+- Display knowledge of Backstage's review process and best practices for plugin design
+- Is supportive of new and occasional contributors and helps get useful PRs in shape to merge
+
+### Privileges
+
+- GitHub code owner of the plugin directory, with rights to approve and merge PRs towards the plugin
+
+### Becoming a Plugin Maintainer
+
+To become a Plugin Maintainer, you first need to be an Organization Member. You can then file a pull request towards [CODEOWNERS.md](./.github/CODEOWNERS) requesting to be added as a code owner of the plugin directory. Existing code owners of that plugin alongside the core maintainers will then review the request.
+
+## Project Area Maintainer
+
+Project Area Maintainers are owners of a particular project area. They are expected to review and merge pull requests towards their area, and also drive development and manage tech health. A Project Area Maintainer also need to commit a certain number of hours per month towards the project, and exercise judgment for the good of the project, independent of their employer. Project Area Maintainers should also mentor new maintainers and participate in and lead community meetings related to their area. New Project Area Maintainers need to be approved by the existing project area maintainers, or the core maintainers if it is a new area.
+
+The maintainers of a project area may be represented by a team in external organization. In this case, the state as a project area maintainer is tied to the membership in that team. New members of the team may automatically be added as maintainers, as well as removed when they leave. This process is governed autonomously by the team of project area maintainers.
```

Comment:
This is highly unusual provision.  Can you explain the reason for it?  What existing community practice does it support?  Are each of the team members required to individually qualify at PAMs?  If not, why not?

Reply:
Aha, I see the comment above.  So for this you just need to make it clear what project qualifications "team members" need to meet.  As in, do they each individually need to qualify as PAMs, or do they meet some other standard (among other things they'll need to agree to the CoC and DCO).

Reply:
The long-term problem this creates for Backstage is that it means that specific Project Areas will be effectively off-limits for contributors who don't work for specific companies.  Only you can judge how much of a headache that's likely to be, but you might want to explicitly give the CMs/EUSs authority to handle that.

Reply:
Yep, we'll definitely need to keep an eye on this setup. If course individual PAM team members still need to agree to CoC and DCO, I'll have a read through again and make sure that's clear.

I feel we do give CMs authority to handle a pretty broad set of conflicts in this space. In particular the PAM requirement both to "demonstrate knowledge of their project area, and how it fits into the larger Backstage project", as well as being "able to exercise judgment for the good of the project, independent of their employer, friends, or team".

The tool there would be the involuntary removal or demotion by CMs. I don't feel we need anything explicit in the governance that CMs should be able to override PAMs decisions, it should be enough to resolve that through conversation.